main.tf

```diff
@@ -1,8 +1,8 @@
+# ------------------------------------------------------------------------------
```

Comment:
I think this file should be renamed to `caller_identity.tf` now that this is all that's in this file. This is done in other projects such as https://github.com/cisagov/cool-sharedservices-openvpn/blob/develop/caller_identity.tf

Reply:
Done in https://github.com/cisagov/assessment-data-import-terraform/pull/4/commits/96aeb374e8b5c8616edf8f3a29d5d86a884e7a2a